src/AppBundle/Controller/Pa/TeamController.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace AppBundle\Controller\Pa;
+
+use AppBundle\Entity as EntityDir;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Component\HttpFoundation\Request;
+use AppBundle\Controller\AbstractController;
+use AppBundle\Form as FormDir;
+
+/**
+ * @Route("/pa/team")
+ */
+class TeamController extends AbstractController
+{
+    /**
+     * @Route("/add-user", name="pa_team_user")
+     * @Template
+     */
+    public function addUserAction(Request $request)
+    {
+        $form = $this->createForm(new FormDir\User\UserDetailsPaType());
```

Comment:
the radio box "Grant user administrator access" is needed in this page. What do yo think about another form inheriting this one ?